source/AI.cpp

```diff
@@ -4005,7 +4005,7 @@ void AI::MovePlayer(Ship &ship, const PlayerInfo &player, Command &activeCommand
 			// If the player is moving slowly over an object, then the player is considering landing there.
 			// The target object might not be able to be landed on, for example an enemy planet or a star.
 			const bool isTryingLanding = (ship.Position().Distance(object.Position()) < object.Radius() && isMovingSlowly);
-			if(object.HasValidPlanet() && object.GetPlanet()->IsAccessible(&ship))
+			if(object.HasValidPlanet() && object.GetPlanet()->IsAccessible(&ship) && object.DistanceAlpha(ship.Position()) > 0.)
```

Comment:
feels to me like it should be a bool and should be clearer no?
like I fr dont know whats going on haha

at the very least gimme a comment on the method

Reply:
I added a new function called isVisible